recipes/pysteps/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "pysteps" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a2443ed3cd1636e60622a320cf7e7e8a0e857ad456e034ef19d084545ad1bc0e
+
+build:
+  number: 0
+  skip: True  # [py<36]"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - numpy
+    - cython
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - cython
+    - pip
+    - openmp 4.0.*  # [osx]
+    - attrdict
+    - jsmin
+    - scipy
+    - matplotlib
+    - jsonschema
+  run:
+    - python
+    - openmp 4.0.*  # [osx]
+    - attrdict
+    - jsmin
+    - jsonschema
+    - matplotlib
+    - netCDF4
+    - numpy
```

Are you using the numpy C bindings? If yes, see [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy):
```suggestion
    - {{ pin_compatible('numpy') }}
```
If not, you probably don't need numpy in the `host` section.

---

Hi @chrisburr, yes, we have a Cython extension that needs numpy to run setup.py. Thanks for the fix.